Users/Install.rst

```diff
@@ -83,10 +83,80 @@ To install coala only (without any bears), you can do:
     **Ubuntu Users** - Do not use ``apt-get install coala``
     for installing coala as that is different software.
 
+Installing inside a pipenv
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Pipenv is user-friendly method of managing virtual environments and
+packages.
+You can look at it as a mix of ``pip`` and ``virtualenv``. So you don't have
+to manage them seperately for your projects.
+
+First, assuming you have installed python and pip earlier from
+this guide we'll now install ``pipenv`` by running the following command:
+
+::
+
+    $ pip install pipenv
+
+Now we create a python 3 environment for the project, move to your project
+directory and run the following command:
+
+::
+
+    $ pipenv --three
+
+The above command will create a virtual environment isolated from your system
+global installation.
+
+.. note::
+
+    **Unix based** (OS X and Linux) - you can find the user base binary
+    directory for your pipenv by running ``python -m site --user-base``.
```

Comment:
Does this matter?

Reply:
It's not important for the setup and can be removed if we don't want to give this information to the user.